test/query/shorthand.test.ts

```diff
@@ -432,16 +572,16 @@ describe('query/shorthand', () => {
        assert.equal(str, 'bin(a,q,maxbins=20)');
     });
 
-    it('should return correct fieldDefShorthand string for bin field with maxbins and scale with scaleType log', () => {
+    it('should return correct fieldDefShorthand string for bin field with maxbins and scale with scaleType ordinal', () => {
       const str = fieldDefShorthand({
-        channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, bin: {maxbins: 20}, scale: {type: ScaleType.LOG}
+        channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, bin: {maxbins: 20}, scale: {type: ScaleType.ORDINAL}
```

Comment:
Bin with ordinal scale?  We don't support it.   I don't understand why you just don't put an obviously valid one like LINEAR here.